weather_mv/loader_pipeline/ee.py

```diff
@@ -405,6 +405,8 @@ def process(self, uri: str) -> t.Iterator[AssetData]:
                                                    ('start_time', 'end_time', 'is_normalized'))
             dtype, crs, transform = (attrs.pop(key) for key in ['dtype', 'crs', 'transform'])
             attrs.update({'is_normalized': str(is_normalized)})  # EE properties does not support bool.
+            # Make attrs EE ingestable.
+            attrs = comb_attributes(attrs)
```

Comment:
optional: could we call this `parse_attributes`?

Reply:
do we use `attrs`? Below, it looks like we use the attributes on `da`.

Reply:
Yes, we use it refer to [Line: 432](https://github.com/google/weather-tools/pull/275/files#diff-5ddebac011830eb7d97f9a4badf76f3a8f2990918c89b70150b585fe3df0a527R432).

Basically `ds.attrs` are used for setting tiff/dataset level metadata were as `da.attrs` are used for setting band level metadata.

Reply:
umm should we rename it to `make_attrs_ee_compatible` -- this makes more sense as per what method is actually doing. WDYT ?